backend/tests/unit/services/prompt/test_prompt_service.py

```
@@ -104,14 +104,26 @@ async def test_create_prompt_success(self, prompt_service, mock_file_ops_service
+                    with patch.object(prompt_service, 'get_prompt') as mock_get:
+                        # Mock the get_prompt call to return the saved prompt
+                        mock_get.return_value = PromptMeta(
+                            prompt=sample_prompt_data,
+                            repo_name=repo_name,
+                            file_path=file_path,
+                            recent_commits=[],
+                            pr_info=None
+                        )
+                        mock_workflow.return_value = None
+                        
+                        result, pr_info = await prompt_service.save_prompt(
```

Ruff (0.14.1)

Unpacked variable `pr_info` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)